clients/src/main/java/org/apache/kafka/common/errors/ReplicaNotAvailableException.java

```diff
@@ -16,7 +16,13 @@
  */
 package org.apache.kafka.common.errors;
 
-public class ReplicaNotAvailableException extends ApiException {
+/**
+ * The replica is not available for the requested topic partition. This may be
+ * a transient exception during reassignments. From version 2.6 onwards, Fetch requests
+ * and other requests intended only for the leader of the topic partition return
```

Comment:
Should it be `leader or follower` her and in the ReplicaNotAvailable message?

Reply:
@ijuma Thanks for the review, updated.